src/pages/api/axiosInstance.ts

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const instance = axios.create({
```

Comment:
이렇게 타입을 잡아주시면 더 좋습니다~
```suggestion
const instance:AxiosInstance = axios.create({
```

Reply:
개인적으로는 어떤 인스턴스인지 표현해줘도 좋을 것 같습니다~

Reply:
저도 좋은 생각이라고 생가합니다.
멘토님이 pr남긴것 수정하면 approve하겠습니다!